content/en/scale-agent/release-notes/agent-plugin/agent-plug-v-0-8-67.md

```diff
@@ -0,0 +1,8 @@
+---
+title: v0.8.67 Armory Agent Clouddriver Plugin (2022-09-13)
+toc_hide: true
+version: 00.08.67
+
+---
+
+Fix missing cluster deletion when removing all associated deployments; not allowing to delete an application because the existence of that empty cluster.
```

Comment:
Suggested edit:

_Fix to ensure cluster deletion when all associated deployments are removed. This fixes an issue where an application could not be deleted due to an empty cluster._